Makefile

```diff
@@ -827,6 +827,9 @@ else
 ifeq ($(findstring s390,$(UNAME_M)),s390)
 DESTCPU ?= s390
 else
+ifeq ($(findstring arm64,$(UNAME_M)),arm64)
+DESTCPU ?= arm64
+else
 ifeq ($(findstring arm,$(UNAME_M)),arm)
 DESTCPU ?= arm
 else
```

Comment:
```suggestion
ifeq ($(findstring arm,$(UNAME_M)),arm)
DESTCPU ?= arm
else
ifeq ($(findstring arm64,$(UNAME_M)),arm64)
DESTCPU ?= arm64
else
```

Reply:
Putting the ifeq `arm` before `arm64` doesn't work because it matches `arm` first and never matches `arm64`

Reply:
oh, sorry. it's a bit unfortunate that this condition cannot be closer to the other one for arm64 but that's fine